docs/metrics-modeling/metrics-microservice-deetsmeter.md

```
@@ -369,3 +369,41 @@ Production code (existing):
+Docker's --env-file takes exactly the KEY=VALUE format your _metricbot.env already is, so you point at it directly:
+
+  cd C:\Projects\sports-data
+  docker build -f src/metrics-modeling/Dockerfile -t sportsdatametricbot:local .
+
+  docker run --rm -p 8080:8080 --env-file "D:\Dropbox\Code\sports-data-provision\_secrets\_metricbot.env" sportsdatametricbot:local
+
+  Then in another terminal:
+
+  curl http://localhost:8080/health
+
+  # a dry-run experiment (no POST, returns the metadata)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)